env.yaml

```diff
@@ -0,0 +1,7 @@
+name: base
+channels:
+  - conda-forge
+dependencies:
+  - python = 3.10.2
+  - pyyaml = 6.0
+  - lxml = 4.7.1
```

Comment:
Need to add a dependency for curl, as this is used to upload the files to the Webin upload area (staging area) prior to the submission phase.
- curl = 7.71.1

Reply:
The latest version of curl is 7.81.0, slightly higher. Hope that's OK with you?